packages/instrumentation-mcp/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## [0.1.0] - Unreleased
+
+### Added
+
+- Initial release of MCP (Model Context Protocol) instrumentation
```

Comment:
We usually add the PR link and title in the changelog, not the specific details , Ex:

Add OpenTelemetry instrumentation for MCP SDK [3186](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/3186)